server/clearbit/enrich.js

```diff
@@ -127,9 +127,12 @@ export function shouldEnrich(message = {}, settings = {}) {
 
   const cbId = user["traits_clearbit/id"];
   const fetched_at = user["traits_clearbit/fetched_at"];
+  
+  // Enrich if we have no clearbit data. Skip if we already tried once.
+  if (fetched_at) return "Fetched_at already set"
```

Comment:
Let's return false and log what happens here. Mixed types in return values can be very confusing